docs/platform/howto/change-billing-contact.rst

```diff
@@ -0,0 +1,28 @@
+Change billing contact
+======================
+
+For projects using credit card billing, the billing contact is automatically set to the user who owns the project's credit card. This means that the billing contact is set to the user who added the payment card and assigned it to project(s). You cannot remove the billing contact from the project. However, you change the billing contact user. 
+
+To change the billing contact user, you need to add a new payment card to the user account you want to assign as the billing contact user. The new payment card can be the same as the one currently used. 
+
+
+Add a new payment card
+~~~~~~~~~~~~~~~~~~~~~~
+To add a new payment card and change the billing contact: 
+
+-  In the `Aiven web console <https://console.aiven.io/>`_, click the **User information** icon on the top right of the screen to view the User profile. 
+  
+.. image:: /images/platform/howto/billing_user_information.png
+
+-  | Next, click the **Payment Options** tab.
+
+.. image:: /images/platform/howto/billing_payment_options.png
```

Comment:
Do you think that the user needs a screenshot to help them click on the "Payment Options" tab, or could we manage without this one? Every screenshot needs updating every time we change the UI so we are moving to include them only when they really illustrate something that is difficult to understand from the text. I'm pretty sure we don't need three, but have another look and see what you think.

Reply:
I don't think this screenshot is required. The text explanation is enough. I will remove this screenshot.